hadoop-ozone/tools/src/main/java/org/apache/hadoop/ozone/debug/DBScanner.java

```diff
@@ -318,7 +319,8 @@ private void printAppropriateTable(
         }
         ManagedRocksIterator iterator;
         if (containerId > 0 && dnDBSchemaVersion != null &&
-            dnDBSchemaVersion.equals("V3")) {
+            dnDBSchemaVersion.equals("V3") &&
+                parent.getDbPath().contains(OzoneConsts.CONTAINER_DB_NAME)) {
```

Comment:
Store `schemaV3` as below.  It can be passed to `displayTable(..)` later on.
```java
        final boolean schemaV3 = dnDBSchemaVersion != null &&
            dnDBSchemaVersion.equals("V3") &&
            parent.getDbPath().contains(OzoneConsts.CONTAINER_DB_NAME);
```